### DISCUSSION TITLE
Arctos Geography Committee Scheduling

### DISCUSSION THREAD

**Discussion by genevieve-anderegg** [ArctosDB/arctos#6765] [General]
Attention all Arctos users that are interested in Geography and Locality data! The Arctos Working Group would like bring back monthly geography meetings, with me as organizer and leader. 
**If you are interested in attending and contributing to a geography data based discussion, please fill out this When2Meet poll with what times on what days of the week would work for you**: https://www.when2meet.com/?21555085-zup1c Please enter your times for Pacific Coast timezone.

Here is the [Geography in Arctos](https://github.com/ArctosDB/arctos/projects/2) project for relevant issues, feel free to add anything issues you would like to discuss. 
Keep an eye out for another announcement when we land on a day and time! Discussions will take place in the Arctos Zoom Room.

**Comment by Jegelewicz** [comment]
@genevieve-anderegg I filled in my general availability. I may have other times open depending upon the week of the month....

**Comment by happiah-madson** [comment]
@genevieve-anderegg I also have. My times are in EST which I wasn't sure was captured by the poll.

**Reply by genevieve-anderegg** [reply]
Thank you, woops I totally forgot about time zones. Would you be able to edit your availability to reflect Pacific Coast Time? That's what Arctos usually uses because of Berkeley.

**Reply by happiah-madson** [reply]
absolutely! my bad!

**Reply by happiah-madson** [reply]
updated!

**Comment by mkoo** [comment]
I added mine as well but have a little flexibility depending on the week of the month-- I assume we are looking for a monthly time.
thx! Agenda to come

**Comment by atrox10** [comment]
I can meet any of the times I wrote, any of those days of the month. I just
picked times that others had already picked as good.

On Tue, Sep 26, 2023 at 6:31 AM Michelle Koo ***@***.***>
wrote:

> I added mine as well but have a little flexibility depending on the week
> of the month-- I assume we are looking for a monthly time.
> thx! Agenda to come
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ArctosDB/arctos/discussions/6765#discussioncomment-7112861>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABCJF4OIHX7OO6STO3IIXVDX4LKJXANCNFSM6AAAAAA5GO2JHU>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Carol L. Spencer, Ph.D.
Staff Curator of Herpetology & Researcher
Museum of Vertebrate Zoology
3101 Valley Life Sciences Building
University of California, Berkeley, CA, USA 94720-3160
***@***.***
510-643-5778
https://mvz.berkeley.edu/mvzherp/

For sending packages via courier (FedEx, UPS, etc), use this:
Carol Spencer
163 Weill Hall
University of California
Berkeley, CA 94720

**Comment by genevieve-anderegg** [comment]
@Jegelewicz Looks like 1-2PM Pacific on Mondays (week of Issues meeting) works for everyone that filled out the poll (only 7 people, but hey it's a time that works for 7/7 people!). Shall we go ahead and schedule?

**Reply by Jegelewicz** [reply]
Now for the weirdness - the first Monday of November is actually the week AFTER the issues meeting. So should the first meeting be October 30 or November 6?

I say pick November 6 and say the Committee is meeting the first Monday of each month. Trying to schedule so that the meetings are "Monday the week of Issues Meetings" will get a bit nuts....

**Comment by genevieve-anderegg** [comment]
That sounds good to me! Definitely more simple!

**Reply by Jegelewicz** [reply]
OK - I added the event to the Arctos calendar and invited you me and Michelle. Who else should get the invite?

**Reply by genevieve-anderegg** [reply]
Phyllis Sharp, Hannah Appiah-Madson, Rosie Poulin, Carol Spencer, Tom Giermakowski

**Reply by mkoo** [reply]
Thanks! @genevieve-anderegg

**Comment by sharpphyl** [comment]
Just noticed that the January meeting would be on the 1st - a holiday - so we'll probably have to adjust that date.

**Reply by Jegelewicz** [reply]
Added to the meeting agenda.
